experimenter/experimenter/targeting/constants.py

```diff
@@ -1734,6 +1729,23 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+SHOPPING_OPTED_IN = NimbusTargetingConfig(
+    name="Users opted in to shopping",
+    slug="shopping_opted_in",
+    description="Users with the shopping experience enabled and who have opted in",
+    targeting="""
+    (
+        ('browser.shopping.experience2023.enabled'|preferenceValue)
+        &&
+        ('browser.shopping.experience2023.optedIn'|preferenceValue == 1)
```

Comment:
I'm pretty sure you only need the second line, as it if its not set it should eval to `undefined` and `undefined == 1` will fail targeting.

Reply:
I misread this and thought it was the same pref, oops ignore me :))